Readme.md

```diff
@@ -0,0 +1,390 @@
+# BillboardTopTen
+
+[BillboardTopTen][heroku] is a web application that allows music enthusiasts to explore Billboard's top ten charts throughout history. Inspired by [Billboard charts][Billboard], BillboardTopTen is built using Ruby on Rails on the backend, React.JS on the Front-end, and a PostgreSQL database.
+
+
+[heroku]: https://billboardtopten.herokuapp.com/
+[Billboard]: http://www.billboard.com/charts
+
+##BillboardTopTen Features
+
+* Dynamic visualization of Billboard's top ten charts for every week
+* Week's top song playing while charts are displayed
+* Smooth music fade in fade out transitions between different charts
+* Date Picker for user exploration of top ten charts through history
+* Genre Picker for user exploration of different music genres
+* Album Images associated with each artist on the chart
+
+
+## BillboardTopTen Walk-through
+
+### BillboardTopTen Web Scraping and Search APIs
+
+BillboardTopTen depends on web scraping HTML form [Billboard charts][Billboard] to retrieve all of the charts. A Python script using [billboard.py][billboardpy] API for accessing music charts was written to scrape ten songs for each chart. Each song in a chart contained the following fields: title, artist, weeks, rank, spotifyId, and spotifyLink.
+
+[billboardpy]: https://github.com/guoguo12/billboard-charts
+
+#### Sample Web Scraping Script Snippet
+
+```python
+import billboard
+from time import sleep
+
+f = open('edm.txt', 'w')
+
+i = 0
+chart = billboard.ChartData('dance-electronic-songs')
+while chart.previousDate:
+    print chart.date
+    if len(chart) < 10:
+        chart = billboard.ChartData('dance-electronic-songs', chart.previousDate)
+
+    f.write("*****")
+    f.write("\n")
+    f.write(chart.date)
+    f.write("\n")
+
+    for x in range(0, 10):
+        f.write(str(chart[x].title))
+        f.write("\n")
+
+        f.write(str(chart[x].artist))
+        f.write("\n")
+
+    ...
+```
+Once all of the charts were scraped, a second script was used to parse data into a JSON. This JSON contains a large hash map with a chart's date as a key, and a value as a list top ten tracks.
+
+From the new JSON, another script was used to loop through the hash map and build queries which where used to retrieve track samples from iTunes.
```

Comment:
I'd say:
`We then run another script that to build queries and hit Itunes search API and retrieve tracks' preview URLs`